src/tbp/monty/frameworks/models/evidence_unsupervised_inference_matching.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2025 Thousand Brains Project
+#
+# Copyright may exist in Contributors' modifications
+# and/or contributions to the work.
+#
+# Use of this source code is governed by the MIT
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/MIT.
+
+from tbp.monty.frameworks.models.evidence_matching import MontyForEvidenceGraphMatching
+
+
+class MontyForUnsupervisedEvidenceGraphMatching(MontyForEvidenceGraphMatching):
+    """This class removes explicit reset logic in unsupervised inference experiments."""
```

Comment:
Similarly here, I would expand a bit on this docstring. Why does removing the reset make the experiment unsupervised?